server/lib/src/filter.rs

```diff
@@ -1278,6 +1295,7 @@ impl FilterResolved {
                 FilterResolved::Eq(a, v, idx)
             }
             FilterComp::SelfUuid => panic!("Not possible to resolve SelfUuid in from_invalid!"),
+            FilterComp::Invalid(_) => panic!("Not possible to resolve Invalid in from_invalid!"),
```

Comment:
This can be resolved, just return the same as you would in the other resolve handlers.

Reply:
Less so that it can't but that I didn't want to. I'll add it (probably good for testing)